### PR TITLE
Add the HART taxonomy

### PR DESCRIPTION
**Pull request by rsaniei**

### CODE REVIEW

**Comment by dgarijo** [comment]
@rsaniei,
please add a readme file indicating who the maintainer of this w3id will be, and a small description. Thanks in advance!

**Comment by rsaniei** [comment]
@dgarijo I added the readme file.
Thanks!

File: hart/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# HART
+**HART:** A Taxonomy for Risk of AI in the Health Domain
+HART represents the mian concepts related to the AI risks in the health domain.
```
Comment (TallTed) [inline hart/README.md:3]:
```suggestion
HART represents the main concepts related to the AI risks in the health domain.
```

**Comment by dgarijo** [comment]
@rsaniei,
please see the suggestion by @TallTed

**Pull request merged**